src/Modules/Family/RegistrationContainer.js

```diff
@@ -28,15 +28,17 @@ const RegistrationContainer = (props) => {
 
   const getUserToken = async () => {
     const localUserToken = localStorage.getItem('userToken');
-    if (!localUserToken || localUserToken === 'undefined') {
+    const tokenExpiresAt = localStorage.getItem('tokenExpiresAt');
+    if (new Date(tokenExpiresAt) < new Date() || !localUserToken || localUserToken === 'undefined') {
```

Comment:
Not sure if I'm reading this correctly, but in my beta environment I only have 'userToken' in my local storage. I don't have a secondary 'tokenExpiresAt' value.
![image](https://user-images.githubusercontent.com/11682225/89812142-a951f580-db0d-11ea-9926-6a5ef9834613.png)

Reply:
Is this code basically allows three different ways for someone to get a token set in their browser?

Reply:
Yes that's right david. I'm storing tokenExpiresAt value now in  local storage.
first time when a new user lands on registration form page, we will be issuing a new token with expiration to the user.
next if some one lands on same page later, we are checking 
if token is Expired or undefined ?
 issueNewToken
else
 useExistingUserToken

Reply:
Thanks for the explanation @maheshkurma , that is exactly what we want!